src/core/task/return.cc

```diff
@@ -87,4 +195,11 @@ void ReturnValues::legion_deserialize(const void* buffer)
   buffer_size_ = ptr - static_cast<const int8_t*>(buffer);
 }
 
+void register_exception_reduction_op(Runtime* runtime, const LibraryContext& context)
+{
+  auto redop_id = context.get_reduction_op_id(LEGATE_CORE_JOIN_EXCEPTION_OP);
+  auto* redop   = Realm::ReductionOpUntyped::create_reduction_op<JoinReturnedException>();
+  runtime->register_reduction_op(redop_id, redop, returned_exception_init, returned_exception_fold);
```

Comment:
Should this be `Runtime::register_reduction_op` ?

Reply:
d91979381eafbeaa7bbd980b91c7db0251bf70a5